### ISSUE TITLE
Release v2.4.1 removed the hpdf_version.h which makes the compilation of pcl-1.12.1 fails

### ISSUE THREAD

**Issue by mob41** [libharu/libharu#240]
The release v2.4.1 removed the `include/hpdf_version.h` file which makes pcl-1.12.1 CMake's `FindLibHaru` stage fails.

This is the commit causing the change:
https://github.com/libharu/libharu/commit/9c5d08466902116e956fd3675f91b6dda41380ae

```
CMake Error at /usr/lib64/cmake/vtk/FindLibHaru.cmake:16 (file):
  file STRINGS file "/usr/include/hpdf_version.h" cannot be read.
Call Stack (most recent call first):
  /usr/lib64/cmake/vtk/VTK-vtk-module-find-packages.cmake:1167 (find_package)
  /usr/lib64/cmake/vtk/vtk-config.cmake:150 (include)
  cmake/pcl_find_vtk.cmake:25 (find_package)
  CMakeLists.txt:375 (include)


-- Could NOT find LibHaru: Found unsuitable version "..", but required is at least "2.4.0" (found /usr/lib/libhpdf.so)
```

I installed LibHaru from https://archlinux.org/packages/community/x86_64/libharu/

**Comment by mob41** [comment]
I found https://github.com/libharu/libharu/pull/237 stating that the version file is now automatically generated?

**Comment by acmiyaguchi** [comment]
I filed a similar issue in VTK here: https://gitlab.kitware.com/vtk/vtk/-/issues/18665

**Comment by bramton** [comment]
My bad. I've resolved this issue in the 2.4.2 release.
